examples/greeting-apps/docker-compose.yaml

```diff
@@ -86,6 +86,7 @@ services:
       - --config=/configs/beyla-config.yml
     volumes:
       - ./configs/:/configs
+      - ./system/sys/kernel/security:/sys/kernel/security
```

Comment:
Will we have to document this e.g. to allow running Beyla in Kubernetes?

Reply:
Oh this is a very good point. I need to document this. As it stands now, if security isn't there we'll assume we can propagate the context, since the security file is missing. But it could be because the users didn't mount it and the host machine doesn't allow it. I'll follow-up with a PR on this.